src/DetailsView/components/adhoc-issues-test-view.tsx

```diff
@@ -1,21 +1,38 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
+import { ScanIncompleteWarningId } from 'common/types/scan-incomplete-warnings';
+import { DetailsViewSwitcherNavConfiguration } from 'DetailsView/components/details-view-switcher-nav';
+import { ScanIncompleteWarning, ScanIncompleteWarningDeps } from 'DetailsView/components/scan-incomplete-warning';
 import * as React from 'react';
 
 import { NamedFC } from '../../common/react/named-fc';
 import { DetailsListIssuesView, DetailsListIssuesViewDeps, DetailsListIssuesViewProps } from './details-list-issues-view';
 import { TargetPageChangedView } from './target-page-changed-view';
 
-export type AdhocIssuesTestViewDeps = DetailsListIssuesViewDeps;
+export type AdhocIssuesTestViewDeps = DetailsListIssuesViewDeps & ScanIncompleteWarningDeps;
 
-export type AdhocIssuesTestViewProps = DetailsListIssuesViewProps;
+export type AdhocIssuesTestViewProps = DetailsListIssuesViewProps & {
+    deps: AdhocIssuesTestViewDeps;
+    switcherNavConfiguration: DetailsViewSwitcherNavConfiguration;
+    scanIncompleteWarnings: ScanIncompleteWarningId[];
+};
 
 export const AdhocIssuesTestView = NamedFC<AdhocIssuesTestViewProps>('AdhocIssuesTestView', ({ children, ...props }) => {
```

Comment:
nitpick (just because it was not added/changed on this PR): `children` props seems unnecessary here.